aliBuild

```diff
@@ -237,10 +247,10 @@ class HttpRemoteSync:
       else:
         hashList = json.loads(urlopen(hashListUrl).read())
         pkgList = json.loads(urlopen(pkgListUrl).read())
-    except URLError, e:
+    except URLError as e:
```

Comment:
You misunderstood me. I'd like the python 2.4 compatible changes to be in a separate PR from the python 2.6 only ones. This way we can easily revert the latter if we have problems with it.